src/compiler/abstract_compiler.nit

```diff
@@ -1777,6 +1781,25 @@ redef class MClassType
 		end
 	end
 
+	redef var is_c_primitive is lazy do
+		if mclass.name == "Int" then
+			return true
+		else if mclass.name == "Bool" then
+			return true
+		else if mclass.name == "Char" then
+			return true
+		else if mclass.name == "Float" then
+			return true
+		else if mclass.name == "NativeString" then
+			return true
+		else if mclass.name == "NativeArray" then
```

Comment:
Actually, when I read this quickly my eyes automatically assumed that "NativeArray" returned `true`.